src/Component/Themes/DropToSelection.js

```diff
@@ -168,32 +168,32 @@ class CircleWithInfoAnimations extends React.Component {
         })
 
         btn.addEventListener('touchleave', function () {
-            //console.log('btn moving end');
+            console.log('btn moving end');
 
         })
         btn.addEventListener('touchcancel', function () {
-            //console.log('btn moving cancel');
+            console.log('btn moving cancel');
         })
 
     }
 
 
 
     allowDrop(ev) {
-        //alert(JSON.stringify(ev.target.id))
+        alert(JSON.stringify(ev.target.id))
         ev.preventDefault();
     }
 
 
     drag(ev) {
-        //alert(JSON.stringify(ev.target.id))
```

Comment:
You are suppose to remove the comments not uncomment them @harsh3311

Reply:
Sure i will do so

Reply:
Any updates? @harsh3311